src/Services/PlayCountService.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+require_once 'src/Services/DatabaseConnector.php';
+require_once 'src/Models/SongsModel.php';
+require_once 'src/Entities/Song.php';
+require_once 'src/Services/DisplayAlbumsSongs.php';
+require_once 'artist.php';
+
+class UpdatePlayCount {
```

Comment:
Once you've moved the updateDatabase method as per my other comment in this file, I suggest renaming this class and file to DisplayPlayButtonService or something like that